lib/open_api_spex.ex

```diff
@@ -3,7 +3,17 @@ defmodule OpenApiSpex do
   Provides the entry-points for defining schemas, validating and casting.
   """
 
-  alias OpenApiSpex.{OpenApi, Operation, Reference, Schema, SchemaResolver}
+  alias OpenApiSpex.{
+    OpenApi,
+    Operation,
+    Operation2,
```

Comment:
Until the next major release, `Operation2` would be considered "experimental". Users will be encouraged to try it out to help work out any issues with it. At the next major release, the implementation of `Operation2` would be moved into `Operation`, replacing the older implementation. A new `Operation2` would be created that delegates to the new `Operation`, to maintain backwards-compatibility, and `Operation2` would be marked as deprecated. Then at the next major release after that, it would be removed. What do you think?